app/scripts/background.js

```diff
@@ -988,7 +987,7 @@ export function setupController(
     let badgeColor = BADGE_COLOR_APPROVAL;
 
     if (pendingApprovalCount) {
-      label = BADGE_LABEL_APPROVAL;
+      label = String(pendingApprovalCount);
```

Comment:
Do we want the same max mechanism so it becomes `9+` etc?

Reply:
thx @matthewwalsh0 you are right

Reply:
Done :)